app/handlers/signup_start.rb

```diff
@@ -23,43 +23,6 @@ def handle
     fatal_error(code: :email_in_use, offending_inputs: [:signup, :email]) if email_in_use?
 
     fatal_error(code: :invalid, offending_inputs: [:signup, :email]) if invalid_email?
-
-    # is there a pre_auth_state and it's email is unchanged
-    if existing_pre_auth_state.try(:contact_info_value) == email
-      existing_pre_auth_state.update_attributes(role: signup_params.role)
-      outputs.pre_auth_state = existing_pre_auth_state
-      # pre_auth_state may have been created in session start
-      # and the the confirmation email will not yet have been sent
-      deliver_validation_email if existing_pre_auth_state.confirmation_sent_at.nil?
-      return
-    end
-
-    # Create a new one
-    new_pre_auth_state = PreAuthState.email_address.create(
-      is_partial_info_allowed: false,
-      contact_info_value: email,
-      role: signup_params.role,
-      signed_data: existing_pre_auth_state.try!(:signed_data),
-      return_to: options[:return_to]
-    )
-
-    # Blow away the user's existing signup email, if it exists
-    existing_pre_auth_state.try(:destroy)
-
-    transfer_errors_from(new_pre_auth_state,
-                         { map: { contact_info_value: :email },
-                           scope: :signup },
-                         true)
-
-
-    outputs.pre_auth_state = new_pre_auth_state
-    deliver_validation_email # Send the pin
-  end
-
-  def deliver_validation_email
-    SignupConfirmationMailer.instructions(
-      pre_auth_state: outputs.pre_auth_state
-    ).deliver_later
```

Comment:
Because pre auth states are being created today on production, we should make sure that they (and their associated code) really really aren't needed.

Reply:
This handler is only being called here (which is currently not how users signup for an account): https://github.com/openstax/accounts/blob/ca7aa5445eaa91d78aaa6e05ab6b4582b228b8c7/app/controllers/legacy/signup_controller.rb#L27


SignupConfirmationMailer is [not sent by anything else](https://github.com/openstax/accounts/search?q=SignupConfirmationMailer) in the code.

Reply:
This is the rabbit hole of PR#1070 😂 

This file needs to be removed.. it's confusing.